client/cliente2e/README.md

```diff
@@ -0,0 +1,26 @@
+# Client E2E Testing
+
+Client e2e test pkg `experimentalclient/` .
+
+Just run:
+```
+./client/cliente2e/test
+```
+
+We will cover more details below.
+
+## Build image
+
+Currently only maintainers of etcd operator have access to test image.
```

Comment:
`maintainers of etcd operator` is strictly longer than `etcd operator maintainers`